docs/src/app/(public)/(content)/production-error/[code]/PageContent.mdx

```diff
@@ -0,0 +1,18 @@
+# Production error
+
+{/**
+@typedef Props
+@property {string} msg The raw error message.
+**/}
+
+<Subtitle>Explanation for minified production error message.</Subtitle>
+<Meta
+  name="description"
+  content="In the production build, error messages are minified to keep your application lightweight."
+/>
+
+A minified Base UI error occurred in the production build of React.
+
+The full error message:
+
+<ErrorDisplay msg={props.msg} />
```

Comment:
Current https://deploy-preview-1463--base-ui.netlify.app/production-error/1

<img width="544" height="234" alt="SCR-20250906-ndbn" src="https://github.com/user-attachments/assets/e715539b-9a9e-47ff-9608-6b2e6b30f591" />

I doubt this is how this should look. It feels like we should render the error red `#ff0000`, like people will see the error in their console first. This feels better: https://react.dev/errors/130.

Reply:
The design can be iterated on by us as well as the Base UI team. The PR is more about setting up the infrastructure for error codes.